framework/src/main/java/org/tron/core/net/TronNetDelegate.java

```diff
@@ -191,6 +195,7 @@ public void processBlock(BlockCapsule block) throws P2pException {
     BlockId blockId = block.getBlockId();
     synchronized (blockLock) {
       try {
```

Comment:
what if there is a exception happen?? In this case, you code will collect this process time, which mean would miss some data????